docs/pages/includes/cloud/cloudmanagedadvisory.mdx

```diff
@@ -0,0 +1,7 @@
+<Details scopeOnly={true} scope={["cloud"]} title="Teleport Cloud">
+This guide deploys the Teleport Auth Service and Proxy Service, which are fully managed in Teleport Cloud. If you are a Teleport Cloud customer, we recommend following our guides for deploying specific services on your Teleport Nodes. Here are some examples:
```

Comment:
```suggestion
This guide deploys the Teleport Auth Service and Proxy Service, which are fully managed in Teleport Cloud. If you are a Teleport Cloud customer, we recommend following our guides for accessing specific resources within your infrastructure. Here are some examples:
```

I wrote this when I mistakenly conflated "Nodes" with "agents," so I've added a suggestion.